components/json/Get_element_by_index/component.yaml

```diff
@@ -0,0 +1,24 @@
+name: Get element by index from JSON
+inputs:
+- {name: Json}
+- {name: Index, type: Integer}
+outputs:
+- {name: Output}
+metadata:
+  annotations:
+    author: Alexey Volkov <alexey.volkov@ark-kun.com>
+implementation:
+  container:
+    image: stedolan/jq:latest
+    command:
+    - sh
+    - -exc
+    - |
+      intput_path=$0
```

Comment:
Is the misspelling in input_path as intput_path intended in all of the files on the PR?

Reply:
Fixed. Thanks.